src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -422,7 +422,13 @@ Whether Harvesting (OAI) service is enabled
         /**
          * Shibboleth affiliation attribute which holds information about the affiliation of the user (e.g. ou)
          */
-        ShibAffiliationAttribute
+        ShibAffiliationAttribute,
+        /*Convert shibboleth AJP attributes from ISO-8859-1 to UTF-8 */
+        ShibAttributeCharacterSetConversionEnabled,
+        /**
+         * Validate physical files for all the datafiles in the dataset when publishing
+         */
+        FileValidationOnPublishEnabled
```

Comment:
Why was FileValidationOnPublishEnabled added? I'm confused.

Reply:
I may have forked from a different branch, where FileValidationOnPublishEnabled was there. I will try a new pull request if I cannot salvage this.

Reply:
I see that FileValidationOnPublishEnabled is in IQSS/develop already, not sure why the diff shows up that way.